tests/conftest.py

```diff
@@ -28,10 +29,15 @@ def pg_dispatcher() -> DispatcherMain:
     return DispatcherMain(BASIC_CONFIG)
 
 
-@pytest_asyncio.fixture(loop_scope="function", scope="function")
+@pytest_asyncio.fixture(
+    loop_scope="function",
+    scope="function",
+    params=[ProcessManager, ForkServerManager],
```

Comment:
This is the key thing I wanted to show here. It just runs the same integration tests but it does it using forkserver. There doesn't seem to be any problems with doing this. They pass. So that's good.